Source/WebCore/css/parser/CSSParserImpl.cpp

```diff
@@ -818,20 +818,16 @@ RefPtr<StyleRuleFontPaletteValues> CSSParserImpl::consumeFontPaletteValuesRule(C
             if (value.isFontFamily())
                 fontFamilies.append(AtomString { value.stringValue() });
         };
-        auto cssFontFamily = properties->getPropertyCSSValue(CSSPropertyFontFamily);
+        RefPtr cssFontFamily = properties->getPropertyCSSValue(CSSPropertyFontFamily);
         if (!cssFontFamily)
             return fontFamilies;
-        auto cssPrimitiveFontFamilies = dynamicDowncast<CSSValueList>(*cssFontFamily);
-        if (cssPrimitiveFontFamilies) {
+        if (RefPtr cssPrimitiveFontFamilies = dynamicDowncast<CSSValueList>(*cssFontFamily)) {
             for (auto& item : *cssPrimitiveFontFamilies)
                 append(downcast<CSSPrimitiveValue>(item));
             return fontFamilies;
         }
-
-        auto cssPrimitiveFontFamily = dynamicDowncast<CSSPrimitiveValue>(*cssFontFamily);
-        if (!cssPrimitiveFontFamily)
-            return fontFamilies;
-        append(*cssPrimitiveFontFamily);
+        if (RefPtr cssPrimitiveFontFamily = dynamicDowncast<CSSPrimitiveValue>(*cssFontFamily))
```

Comment:
`releaseNonNull` instead of `*` perhaps

Also I would name the local something shorter like `family`.